book-worm-2010/book-worm-fe
# Feature/error handling

### What does this PR do? 
>Clearly and concisely describe the feature. 
This PR adds error handling to the forms.

### Is this a feature or Fix/Refactor?  
Feature

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/BookDetails/BookDetails.js` lines: 20-27;
`src/NewBookMarkForm/NewBookMarkForm.js' lines: 63-73, 85-99;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` branch
2. run `npm i`
3. run `npm start`
4. Run through the application
